Adapt to Slint: enable notifications by default and run slapt-src in cron.

diff -ru salix-update-notifier-1.0.0/src/salix-update-notifier-update salix-update-notifier-1.0.0.new/src/salix-update-notifier-update
--- salix-update-notifier-1.0.0/src/salix-update-notifier-update	2025-11-03 04:17:24.000000000 -0500
+++ salix-update-notifier-1.0.0.new/src/salix-update-notifier-update	2026-02-11 12:37:20.886784539 -0500
@@ -10,4 +10,5 @@
 # This script is to be placed inside /etc/cron.hourly or another cron directory
 
 /usr/sbin/slapt-get -u
+/usr/bin/slapt-src -u &
 [ -x /usr/bin/flatpak ] && /usr/bin/flatpak update --noninteractive --appstream -y
diff -ru salix-update-notifier-1.0.0/src/salix-update-notifier.conf salix-update-notifier-1.0.0.new/src/salix-update-notifier.conf
--- salix-update-notifier-1.0.0/src/salix-update-notifier.conf	2025-11-03 04:17:24.000000000 -0500
+++ salix-update-notifier-1.0.0.new/src/salix-update-notifier.conf	2026-02-11 12:35:37.678079303 -0500
@@ -1,2 +1,2 @@
 interval=2h
-notification=false
+notification=true
